src/schemas/Banner/index.ts

```diff
@@ -0,0 +1,91 @@
+import { gql } from 'apollo-server-express';
+import { makeExecutableSchema } from 'graphql-tools';
+import { isEqual } from 'lodash';
+
+type BannerObj = {
+  [key: string]: string
+}
+
+const bannerFieldsRequired: BannerObj = {
+  'dismissable': 'boolean',
+  'id': 'number',
+  'level': 'string',
+  'title': 'string',
+};
+
+const bannerFieldsAllowed: BannerObj = {
+  ...bannerFieldsRequired,
+  'message': 'string',
+};
+
+const bannerLevelsAllowed = [
+  'error',
+  'info',
+  'warning',
+];
+
+const getBanners = () => {
+  try {
+    const bannersStr = process.env.BANNERS || '';
+    const bannersParsed = JSON.parse(bannersStr);
```

Comment:
Can we just use Typescript and type our `bannerKeys` here and have it cover a lot of the error handling?
```
const bannersParsed: Array<Banners>  = JSON.parse(bannersStr)
```